### ISSUE TITLE
sp500-strategies subject

### ISSUE THREAD

**Issue by jarmo-seljamaa** [01-edu/public#2484]
sp500-strategies
---
As with other tasks in AI module, the instructions and audit questions lack proof-reading for spelling and clarity.

> DataFrame with a Machine learning metrics on train et validation sets on all folds of the train set. 

Train et validation?

From the audit:

> [Does the last validation set of the train set not overlap on the test set?](https://github.com/01-edu/public/tree/master/subjects/ai/sp500-strategies/audit#does-the-last-validation-set-of-the-train-set-not-overlap-on-the-test-set)

Possible answers:
- Yes, the last validation set does not overlap on the test set.
- No, the last validation set does not overlap on the test set.
- No, the last validation set does overlap on the test set.

Audit has only singular Yes/No options, negation questions are silly to include.

Also this one:
> Do all of the folds not contain data from the same day? The split should be done on the dates.

Apart from the language problem (negation question + "do *all* not contain" can mean that some may contain?), it seems illogical. The data we are using is for 500 different stocks, so we do have 500 data points from the same day. It seems no problem to include the data from the same day in the folds, because the data is from different tickers. Perhaps there is something here, we're missing. If so, please do explain.

**Comment by nprimo** [comment]
Hi @jarmo-seljamaa, thank you for your feedback! I rephrase the 2 audit questions you pointed out to improve their clarity.

**Comment by jarmo-seljamaa** [comment]
Thanks! However the new wording still contains negation:

> Is the last validation set of the train data not overlapping with the test data?

Possible answers:
- Yes, it is not overlapping.
- No, it is not overlapping.
- No, it is overlapping.

**Comment by nprimo** [comment]
Would the following be clearer?

> Is the last validation set of the train data not overlapping with the test data? Yes (the two sets are not overlapping) or no (the two sets are overlapping).

**Comment by jarmo-seljamaa** [comment]
I've seen this type of solution in some earlier tasks:

> Can you confirm that the last validation set of the train data is not overlapping with the test data?

Not the smoothest, but at least it doesn't have 3 possible answers.
